### DISCUSSION TITLE
Intellisense for cake does not work

### DISCUSSION THREAD

**Discussion by fawesomebuildmachine** [cake-build/cake#4232] [Extension Q&A]
<img width="1033" alt="image" src="https://github.com/cake-build/cake/assets/146713120/f422fbcb-393c-4138-a524-57d6a114d87b">

Intellisense for cake does not work for me, and there is no omniShell select project command

**Comment by devlead** [comment]
OmniSharp isn't on by default since they shipped C# DevKit

Try in your workspace adding settings
  ```json
  "dotnet.preferCSharpExtension": true,
    "dotnet.server.useOmnisharp": true
```

**Reply by BrushkouMatvey** [reply]
<img width="1120" alt="image" src="https://github.com/cake-build/cake/assets/48620777/cd0c793e-d10b-4305-80fe-68214a81c842">

**Reply by BrushkouMatvey** [reply]
<img width="1007" alt="image" src="https://github.com/cake-build/cake/assets/48620777/66faafd5-0e9b-43f0-9ad3-26bdc48db1de">

**Reply by BrushkouMatvey** [reply]
<img width="797" alt="image" src="https://github.com/cake-build/cake/assets/48620777/45737aa9-0f57-473f-9e67-6a14681376dc">

**Reply by BrushkouMatvey** [reply]
<img width="905" alt="image" src="https://github.com/cake-build/cake/assets/48620777/465e3c60-f34d-46f7-9164-cdfb5355d2dd">

I select here Cake build_0 but nothing changes, and when I select it again there is a checkmark next to build_0 sln
